graylog2-web-interface/src/preflight/components/ConfigurationWizard/CAUpload.tsx

```diff
@@ -89,7 +89,7 @@ const CAUpload = () => {
     UserNotification.error('CA upload failed');
   }, []);
 
-  const { mutate: onProcessUpload, isLoading } = useMutation(submitUpload, {
+  const { mutateAsync: onProcessUpload, isLoading } = useMutation(submitUpload, {
```

Comment:
We should also make this change in `CACreateFrom` and `RenewalPolicyConfiguration`. Shall we do it in this PR?

Reply:
Sure! Can you do it in a follow-up PR?

Reply:
Alright!